lib/rubocop/runner.rb

```diff
@@ -254,7 +259,7 @@ def enable_rails_cops(config)
     def mobilized_cop_classes(config)
       @mobilized_cop_classes ||= {}
       @mobilized_cop_classes[config.object_id] ||= begin
-        cop_classes = Cop::Cop.all
+        cop_classes = Cop::Cop.all.to_a
```

Comment:
I could have the `all` method be the thing that invokes `to_a` which would resolve it being called in many places.